app/controllers/relationships_controller.rb

```diff
@@ -1,5 +1,5 @@
 class RelationshipsController < ApplicationController
-  before_action :authenticate_user!
+  before_action :authenticate_user!, except: [:follow]
```

Comment:
Identify_user has the logic of redirecting if user is not signed in. But it won't be run unless we by-pass `authenticate user`. #197 suggests that we should redirect to login page instead of giving an unauthorized error.

I am gonna add a test in PR.